Modelica/Electrical/Analog/Basic.mo

```diff
@@ -1240,10 +1241,10 @@ the user has to allocate the parameter vector <em>L[6] </em>, since <em>Nv=(N*(N
     parameter SI.Current Imaxsi=25.0e-3 "Maximal output current (sink current)";
 
     // number of intervals: 2500, stop time: 0.003
-    parameter SI.Time Ts=0.0000012 "sampling time";
+    parameter SI.Time Ts=0.0000012 "Sampling time";
 
     // constant expressions
-    constant Real Pi=3.141592654;
+    constant Real Pi=3.141592654 "Obsolete constant pi; will be removed in future release";
 
```

Comment:
This is just a marker for a later search of the obsolete keyword.

The obsolete annotation works on classes, not on components.

Reply:
Actually I wonder if this constant can not simply be removed. It seems unlikely that one would use the constant `Pi` from this example for user models. @HansOlsson @svorkoetter, any thoughts form a tool vendor's perspective?

Reply:
>Actually I wonder if this constant can not simply be removed. It seems unlikely that one would use the constant Pi from this example for user models. @HansOlsson @svorkoetter, any thoughts form a tool vendor's perspective?

I agree that just removing it seems simplest - and I hope no-one is using that constant - both because it would be bad style and because pi had an inaccurate value.

Reply:
I would prefer to remove Pi in the next release where we are allowed to break backwards compatibility.

Reply:
Although technically removing it would break backwards compatibility, unless someone explicitly wrote Modelica.Electrical.Analog.Basic.OpAmpDetailed.Pi in their code, this wouldn't happen, so I have no issues with removing it.

Reply:
There's also no harm in leaving it there for now. It is clearly marked as obsolete, so no one should start using it.